cmd/agent/subcommands/run/command.go

```diff
@@ -348,6 +349,7 @@ func getSharedFxOption() fx.Option {
 
 		// workloadmeta setup
 		collectors.GetCatalog(),
+		processcollector.GetFxOptions(),
```

Comment:
This collector was imported separately from `collectors.GetCatalog` due to avoid introducing huge amounts of dependencies in builds this collector will not be used. Namely, this was an issue with the `dogstatsd` build and resulted in a large increase in package size.

Reply:
Is there a better way to handle this issue.

Reply:
This should be a component in itself if it's imported in a main through FX.

Reply:
Will be resolved through https://github.com/DataDog/datadog-agent/pull/28172